opensrp-chw-hf/src/main/assets/json.form/hiv_outcome.json

```diff
@@ -99,11 +99,84 @@
             "test_results_calculation"
           ]
         },
+        {
+          "name": "where_was_test_conducted",
+          "type": "radio_group",
+          "properties": {
+            "text": "Where was the test conducted?"
+          },
+          "meta_data": {
+            "openmrs_entity": "concept",
+            "openmrs_entity_id": "reason_no_action_was_taken",
```

Comment:
This is incorrect, the openmrs_entity_id should be "where_was_test_conducted" in this case